tools/devtools/utils.mjs

```diff
@@ -0,0 +1,262 @@
+
+const blogDest = './site/{{lang}}/blog/new-in-devtools-{{version}}/index.md';
+const outlineDest = './site/_includes/partials/devtools/{{lang}}/whats-new.md';
+const bannerDest = './tools/devtools/_temp/new-in-devtools-banner-{{lang}}.svg';
+
+const translation = {
+  language: {
+    es: 'Spanish',
+    ja: 'Japanese',
+    ko: 'Korean',
+    pt: 'Portuguese',
+    ru: 'Russian',
+    zh: 'Chinese',
+  },
+  title: {
+    en: "What's New in DevTools (Chrome {{version}})",
+    es: 'Qué hay de nuevo en DevTools (Chrome {{version}})',
+    ja: 'DevTools の新機能 (Chrome {{version}})',
+    ko: 'DevTools 의 새로운 기능 (Chrome {{version}})',
+    pt: 'O que há de novo no DevTools (Chrome {{version}})',
+    ru: 'Новинки DevTools (Chrome {{version}})',
+    zh: 'DevTools 新功能（Chrome {{version}}）',
+  },
+  thankful: {
+    es: '*Gracias  por la traducción [Miguel Ángel](https://midu.dev) y por la revisión [Carlos Caballero](https://carloscaballero.io).*',
+    ja: '*翻訳者の [technohippy](https://github.com/technohippy) さん、レビュアーの [yoichiro](https://github.com/yoichiro) さん、 [lacolaco](https://github.com/lacolaco) さん、 [yoshiko-pg](https://github.com/yoshiko-pg) さんに感謝いたします。*',
+    ko: '*이 게시글의 번역에는 [최원영](https://www.linkedin.com/in/toruchoi)님이 참여하셨으며, [조은](https://developers.google.com/community/experts/directory/profile/profile-eun-cho)님과 [도창욱](https://developers.google.com/community/experts/directory/profile/profile-changwook-doh)님이 리뷰를 맡아 주셨습니다.*',
+    pt: '*Tradução realizada por [Alvaro Camillo Neto](https://www.linkedin.com/in/alvarocamillont/) . Revisão por [Lucas Santos](https://lsantos.dev).*',
+    ru: '*Переводы предоставлены [Alena Batitskaia](https://twitter.com/ABatickaya). Редактор — [Maxim Salnikov](https://twitter.com/webmaxru).*',
+    zh: '*感谢 [Poong Zui Yong](https://www.linkedin.com/in/zui-yong-poong-1b507b14/) 提供的翻译*',
+  },
+  reviewers: {
+    es: ['midudev', 'Caballerog', 'and-oli'],
+    ja: ['yoshiko-pg', 'lacolaco', 'technohippy', 'yoichiro'],
+    ko: ['techhtml', 'TORU0239', 'cwdoh'],
+    pt: ['alvarocamillont', 'khaosdoctor'],
+    ru: ['solarrust', 'webmaxru', 'kateryna-prokopenko'],
+    zh: ['xyugroup', 'aquaMAX', 'liuliangsir', 'louisyoong', 'hanselfmu'],
+  },
+  banner: {
+    en: "What's new in",
+    es: 'Qué hay de nuevo en',
+    ja: '新機能',
+    ko: '새로운 기능',
+    pt: 'O que há de novo no',
+    ru: 'Новинки',
+    zh: '新功能',
+  },
+};
+
+/**
+ * Get Today's date in ISO 8601
+ * @returns YYYY-MM-DD
+ */
+function getToday() {
+  return new Intl.DateTimeFormat('fr-CA', {
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit',
+  }).format(Date.now());
+}
+
+/**
+ * Generate What's New in DevTools blog post banner
+ * @param {String} version - release version
+ * @param {String[]} langs - languages you want to generate
+ */
+export async function createWndtBanners(version, langs) {
+  for (const lang of langs) {
+    const output = nunjucks.render(bannerTemplate, {
```

Comment:
Inspired by your previous review comment, I change most of the regex replace syntax with [nunjucks render](https://mozilla.github.io/nunjucks/api.html#render). 

I can't use 11ty syntax as I need to pre-render it beforehand.